src/JitInterface/src/CorInfoTypes.cs

```diff
@@ -1180,139 +1180,22 @@ public struct ILVarInfo
         public uint varNumber;
     };
 
+    [StructLayout(LayoutKind.Sequential)]
     public struct NativeVarInfo
     {
-        public uint startOffset;
-        public uint endOffset;
-        public uint varNumber;
-        public VarLoc loc;
+        public UInt32 startOffset;
```

Comment:
Framework coding conventions (that we follow) prefer the C# aliases (`uint`, `int` to `UInt32`, `Int32`) when declaring variables or fields.

(Applies here and everywhere else.